src/Bootloader/CycleOrmBootloader.php

```diff
@@ -51,10 +50,12 @@ public function __construct(
     public function boot(
         Container $container,
         FinalizerInterface $finalizer,
-        EnvironmentInterface $env
     ): void {
         $finalizer->addFinalizer(
             static function () use ($container): void {
+                if ($container->hasInstance(EntityManagerInterface::class)) {
+                    $container->get(EntityManagerInterface::class)->clean();
```

Comment:
This line should be covered by tests

Reply:
@kafkiansky Hi, could you please cover this line with tests and we will merge the PR!

Reply:
@kafkiansky Hi! Could you please provide a the following test:

Something like below:

1. Add an entity to Entity manager for update/delete
2. Run finalizer 
3. Try to run Entity manager method  `run` 
4. Check if the Entity wasn't update/delete

Reply:
Hi, done.